### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

File: mingw-w64-meson/PKGBUILD [hunk 1]
```diff
@@ -4,14 +4,15 @@ _realname=meson
...
-pkgrel=1
+pkgrel=2
 pkgdesc="High-productivity build system (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
 url="https://mesonbuild.com/"
 license=("Apache 2")
 options=('strip' 'staticlibs')
 depends=("${MINGW_PACKAGE_PREFIX}-python"
+         "${MINGW_PACKAGE_PREFIX}-python-setuptools"
```
Comment (lazka) [inline mingw-w64-meson/PKGBUILD:15]: hm, it shouldn't need it

Comment (Biswa96) [reply mingw-w64-meson/PKGBUILD:15]: Found this `from pkg_resources import load_entry_point` in `meson-script.py`.

Comment (lazka) [reply mingw-w64-meson/PKGBUILD:15]: This is only used with Python <= 3.7

Comment (Biswa96) [reply mingw-w64-meson/PKGBUILD]: Python version sucks!

**Comment by lazka** [comment]
somehow this didn't get marked as merged? closing manually.

**Pull request closed without merging**